Kitodo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -124,6 +124,10 @@
     <filter>
         <filter-name>requestControlFilter</filter-name>
         <filter-class>de.sub.goobi.helper.servletfilter.RequestControlFilter</filter-class>
+        <init-param>
+            <param-name>excludePattern</param-name>
+            <param-value>.*\.(js|css)\.jsf</param-value>
```

Comment:
Thank you for clarification this. Without this context information the exclude pattern looks strange. If this change where inside a primefaces pull request and not a stand alone pull request I would not asked for this as currently there are no or only a few files with this kind of file ending.